configs/milvus.yaml

```diff
@@ -482,6 +482,8 @@ dataCoord:
     serverMaxRecvSize: 268435456
     clientMaxSendSize: 268435456
     clientMaxRecvSize: 536870912
+  sync:
```

Comment:
This "sync" can be easily confused with persistence 'sync'. It is recommended to rename it to datacoord.segment.syncSegmentsInterval or simply datacoord.syncSegmentsInterval.